webpay/api/tests/test_api.py

```diff
@@ -132,6 +133,12 @@ def test_configures_transaction_fail(self):
         res = self.post(req='')  # cause a form error.
         eq_(res.status_code, 400)
 
+    def test_unsupported_jwt_algorithm(self):
+        with self.settings(SUPPORTED_JWT_ALGORITHMS=['HS256']):
+            res = self.post(
+                request_kwargs={'jwt_kwargs': {'algorithm': 'none'}})
+        eq_(res.status_code, 400)
```

Comment:
oh yeah, I forgot this is a standard JSON API response so we have a readable error code. Thanks.